ui/app/src/utils/state.ts

```diff
@@ -359,6 +360,20 @@ export function getStoredBrowseDialogCompactMode(username: string): boolean {
   return JSON.parse(localStorage.getItem(`craftercms.${username}.browseDialog.compactMode`)) ?? false;
 }
 
+export function setStoredBrowseDialogViewMode(username: string, mode: MediaCardViewModes): void {
+  localStorage.setItem(`craftercms.${username}.browseDialog.viewMode`, mode);
+}
+
+export function getStoredBrowseDialogViewMode(username: string): MediaCardViewModes {
+  // The viewMode field used to be compactMode (boolean). For backwards compatibility, if the viewMode is not set and
+  // compactMode exists, set value accordingly.
+  const backwardsCompatibilityValue = getStoredBrowseDialogCompactMode(username) ? 'compact' : 'card';
+  return (
+    (localStorage.getItem(`craftercms.${username}.browseDialog.viewMode`) as MediaCardViewModes) ??
+    backwardsCompatibilityValue
+  );
+}
```

Comment:
Let's clean up.
- Remove getStoredBrowseDialogCompactMode
- Check for the old key in the new storing modes and if present delete and write the new.
  - The setted should delete the old and write the new
  - The getter must check the old and if not present go to the new
  - Add a clean up todo for later versions to fully remove old key
- Adjust the Account preference clean up stuff to include all this stuff too